tests/x11/yast2_lan_restart.pm

```diff
@@ -90,9 +90,13 @@ sub run {
     unless (is_network_manager_default) {
         # Run detailed check only if explicitly configured in the test suite
         check_etc_hosts_update() if get_var('VALIDATE_ETC_HOSTS');
-        verify_network_configuration(\&check_network_card_setup_tabs);
+        record_info "check_network_card_setup_tabs";
+        verify_network_configuration(\&check_network_card_setup_tabs, 'reload');
```

Comment:
Previsouly, we have checked that no restart happens, does reload happen in all cases (e.g. SLE 15 SP2 vs SP1)?

Reply:
short answer is yes. 
Please take a look to the VR. it is happening in both cases.
i have already explained this to a previous comment. Because there was not filter to catch this, with the result the reload to not be seen at all. in the old code there was only restart or not restart.

Reply:
How can you assume reload will happen on `<=15-sp1` when there is no reload but restart ?

Reply:
And there is no restart expected on <=15-sp1. https://fate.suse.com/318787

Reply:
Even better, looks like 15-sp1 is using reload https://openqa.suse.de/tests/4131624#step/yast2_lan_restart/94

Reply:
As there are tons of changes, I cannot be sure that VR in the description matches latest code. In the http://aquarius.suse.cz/tests/2406#step/yast2_lan_restart/35 I see `no_restart` checks as per info boxes, and you check reload now. Also, there are 3 other modules `tests/yast2_gui/yast2_lan_restart_*` which use check_network_status with `no_restart` which will end up in `die` with your current code.

Reply:
> How can you assume reload will happen on <=15-sp1 when there is no reload but restart ?

I found this during the testing. that for <=15-sp1 had a reload which was not checked before as the checks considered only restart and no_restart.

> And there is no restart expected on <=15-sp1. https://fate.suse.com/318787

Can you elaborate here. for what service. because on <=15-sp1 occurs only for `change_hw_device_name`

> As there are tons of changes, I cannot be sure that VR in the description matches latest code. In the http://aquarius.suse.cz/tests/2406#step/yast2_lan_restart/35 I see no_restart checks as per info boxes, and you check reload now. Also, there are 3 other modules tests/yast2_gui/yast2_lan_restart_* which use check_network_status with no_restart which will end up in die with your current code.

latest run is http://aquarius.suse.cz/tests/2419. 
i didnt notice the other files. they affected only by last change with renames the 'no_start' to 'no_start_or_reload'. I will take care of it

Reply:
Looks like sle 15-SP1 is doing also reload but not everywhere as sle15-SP2 ... mess
sle <=15 (sle12*) don't do reload at all, only restart

Reply:
Somebody had to find out if the new reload behaviour is only on sle15-sp2. it was not me.

Reply:
so is it ok to remove the expected_status from `check_network_card_setup_tabs` and leave it fails. from what i understand what you expect `check_network_card_setup_tabs` should be _no_restart_or_reload_

Reply:
@b10n1k yes no_restart_or_reload for <=15, 15-sp1 & 15sp2 do reload here, right ?